.github/workflows/prettier.yml

```diff
@@ -10,13 +10,17 @@ jobs:
       - uses: actions/setup-node@v2
         with:
           node-version: '14'
+      - run: npm ci
+      - name: Fail CI
+        if: ${{ github.event_name == 'pull_request' || github.event_name == 'pull_request_target' }}
+        run: npx prettier --check public/data/*.json
+        env:
```

Comment:
is a token needed to do the check?

Reply:
nope, but I think it is needed for annotations